src/lib/routes/admin-api/user-admin.ts

```diff
@@ -475,4 +533,48 @@ export default class UserAdminController extends Controller {
         await this.userService.changePassword(+id, password);
         res.status(200).send();
     }
+
+    async getUserPats(
+        req: IAuthRequest<{ id: number }>,
+        res: Response<PatSchema | NotFoundError>,
+    ): Promise<void> {
+        if (!this.flagResolver.isEnabled('serviceAccounts')) {
+            throw new NotFoundError('Not enabled');
+        }
+        const { id } = req.params;
+        const pats = await this.patService.getAll(id);
+        this.openApiService.respondWithValidation(200, res, patsSchema.$id, {
+            pats: serializeDates(pats),
+        });
+    }
+
+    async createUserPat(
+        req: IAuthRequest<{ id: number }>,
+        res: Response,
+    ): Promise<void> {
+        if (!this.flagResolver.isEnabled('serviceAccounts')) {
+            throw new NotFoundError('Not enabled');
+        }
+        const { id } = req.params;
+        const pat = req.body;
+        const createdPat = await this.patService.createPat(pat, id, req.user);
+        this.openApiService.respondWithValidation(
+            201,
+            res,
+            patSchema.$id,
+            serializeDates(createdPat),
+        );
+    }
+
+    async deleteUserPat(
+        req: IAuthRequest<{ id: number; patId: number }>,
+        res: Response,
+    ): Promise<void> {
+        if (!this.flagResolver.isEnabled('serviceAccounts')) {
+            throw new NotFoundError('Not enabled');
+        }
+        const { id, patId } = req.params;
+        await this.patService.deletePat(patId, id);
```

Comment:
What happens here if the user is not define? Would love to see some e2e tests validating all happy cases and the edge cases.